zubhub_frontend/zubhub/src/assets/js/styles/views/login/loginStyles.js

```diff
@@ -1,127 +1,188 @@
+    fontWeight: 400,
+    lineHeight: '32px',
+    color: theme.palette.text.secondary,
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '12px',
+      lineHeight: 'unset',
     },
   },
-  customInputStyle: {
-    borderRadius: 15,
-    '&.MuiOutlinedInput-notchedOutline': {
-      border: '2px solid var(--primary-color3)',
-      boxShadow: `rgba(var(--primary-color3), 0.2rem) 0 0 0 0.2rem`,
+  header2: {
+    color: theme.palette.text.primary,
+    fontSize: '30px',
+    fontWeight: 700,
+    lineHeight: '50px',
+  },
+  subHeader2: {
+    color: theme.palette.text.primary,
+    fontSize: '18px',
+    fontWeight: 600,
+    lineHeight: '24px',
+  },
+  text2: {
+    marginBottom: '40px'
+  },
+  wizard: {
+    width: '100%',
+  },
+  outlinedInput: {
+    borderRadius: theme.spacing(1),
+    paddingInline: theme.spacing(3),
+    gap: theme.spacing(3),
+    '& .MuiOutlinedInput-input': {
+      padding: '20px 0',
     },
     '&.MuiOutlinedInput-root': {
       '&:hover fieldset': {
-        border: '2px solid var(--primary-color3)',
-        boxShadow: `rgba(var(--primary-color3), 0.2rem) 0 0 0 0.2rem`,
+        border: `1px solid ${theme.palette.primary.light}`,
       },
       '&.Mui-focused fieldset': {
-        border: '2px solid var(--primary-color3)',
-        boxShadow: `rgba(var(--primary-color3), 0.2rem) 0 0 0 0.2rem`,
-      },
-      [theme.breakpoints.up('1600')]: {
-        fontSize: '1.7rem',
+        border: `2px solid ${theme.palette.primary.light}`,
       },
     },
   },
-  secondaryLink: {
-    color: 'var(--primary-color3)',
-    '&:hover': {
-      color: 'var(--secondary-color6)',
+  formControl: {
+    gap: theme.spacing(3),
+    [theme.breakpoints.down('md')]: {
+      gap: theme.spacing(2),
     },
-    [theme.breakpoints.up('1600')]: {
-      fontSize: '1.2rem',
+    [theme.breakpoints.down('sm')]: {
+      gap: theme.spacing(1),
     },
   },
-  center: {
-    display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'center',
+  inputIcon: {
+    fontSize: theme.spacing(3),
+    fill: theme.palette.grey[400],
   },
-  dividerText: {
-    whiteSpace: 'nowrap',
-    [theme.breakpoints.up('1600')]: {
-      fontSize: '1.2rem',
+  button: {
+    fontSize: '18px',
+    fontWeight: 600,
+    paddingBlock: '20px',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '12px',
+      padding: '5px 16px',
     },
   },
-  divider: {
-    width: '30%',
-    marginRight: '1em',
-    marginLeft: '1em',
-    [theme.breakpoints.up('1600')]: {
-      height: '0.1em',
+  spaceBackground: {
+    width: '100%',
+    height: '100%',
+  },
+  backContainer: {
+    borderRadius: '50%',
+    background: theme.palette.background.default,
+    padding: '10px',
+  },
+  backIcon: {
+    fontSize: theme.spacing(3),
+  },
+  gridControl: {
+    background: theme.palette.background.default,
+    padding: '0 40px',
+    boxShadow: '2px 4px 5px 0px rgba(0, 184, 196, 0.05)',
+  },
+});
+
+export const step1Styles = theme => ({
```

Comment:
step1Styles, step2Styles, step3Styles, step4Styles, step5Styles, should all be inside the styles. These are just form fields and they are to be in the same place